jinja2/includes/icons/social/discourse.svg

```diff
@@ -0,0 +1 @@
+<svg class="icon icon-discourse" xmlns="http://www.w3.org/2000/svg" width="13pt" height="14pt" viewBox="0 0 13 14"><path d="M6.555.875C2.996.875 0 3.57 0 6.895l.004 6.23 6.55-.004c3.563 0 6.446-2.8 6.446-6.125C13 3.672 10.117.875 6.555.875zM6.5 10.5a3.845 3.845 0 0 1-1.578-.332l-2.356.55.668-2.05A3.33 3.33 0 0 1 2.785 7c0-1.934 1.664-3.5 3.715-3.5 2.05 0 3.715 1.566 3.715 3.5S8.55 10.5 6.5 10.5zm0 0"/></svg>
```

Comment:
I would suggest running this through SVGOmg [https://jakearchibald.github.io/svgomg/]. Also, I would recommend removing the `width` and `height` attributes or, replacing the `pt` unit with the equivalent `px` value. Roughly that would be:

```
...  width="17px" height="19px" 
```

One other nit is to ensure there is an empty line at the end of the file. Thanks @Mihir-Karbelkar